src/lib/string.ts

```diff
@@ -1,4 +1,4 @@
-import printj from 'printj'
+import * as printj from 'printj'

```

Comment:
```suggestion
import { sprintf } from 'printj'
```
Can we do it this way instead? and update the uses below?